package.json

```diff
@@ -37,6 +44,15 @@
           ".build"
         ],
         "configuration": "./syntax/language-configuration.json"
+      },
+      {
+        "id": "go",
+        "extensions": [
+          ".go"
+        ],
+        "aliases": [
+          "Go"
+        ]
```

Comment:
I assume this doesn't seem like it interferes with the go extension? You can still run, debug etc. go code with this plugin active?

Reply:
Ya, I checked.